tools/download_paks.py

```diff
@@ -12,17 +12,16 @@
 
 # list all available scene_ids for download
 available_scene_ids = [
+    "Common",
```

Comment:
Can we rename this to `common` so it matches all the other pak files? The filename of each pak file doesn't matter to Unreal, so we wouldn't need to rebuild anything. We would just need to rename the pak files on AWS, and change the default name in `build_paks` so it produces the correct output the next time we need to run it.